packages/grafana-ui/src/components/Stack/Stack.tsx

```diff
@@ -2,17 +2,28 @@ import React from 'react';
 
 import { ThemeSpacingTokens } from '@grafana/data';
 
-import { Direction, Flex } from '../Flex/Flex';
+import { Flex } from '../Flex/Flex';
+
+import { HorizontalStack } from './HorizontalStack';
+import { VerticalStack } from './VerticalStack';
 
 interface StackProps {
-  direction?: Direction;
+  direction?: 'horizontal' | 'vertical';
```

Comment:
Should we keep it as `'row' | 'column'` to be consistent with the props for `Flex`?

Reply:
If we abstract it away by having two components for the directions, then I guess it doesn't matter. I think `row` and `column` is a bad way to describe direction. Also, for the users of this component, the underlying implementation should be abstracted away, and I think this is one way to make it more clear.